### ISSUE TITLE
reduce whitespace

### ISSUE THREAD

**Issue by howardchung** [yasp-dota/yasp#616]
Comment about excessive whitespace on Reddit.  Maybe we can try applying a global style where we drastically reduce the cell-padding?

**Comment by howardchung** [comment]
not actually sure there's a whole lot of good places to target.

One is the large amount of horizontal whitespace in the Farm/Distribution table.

**Comment by howardchung** [comment]
Feedback:
https://www.reddit.com/r/DotA2/comments/3kcgla/yasp_source_2_ads/cuwk4qi

**Comment by howardchung** [comment]
* We can shrink the height of the image in the hero_td cell in order to reduce vertical row whitespace.

* Last row having same color: Intentional, we added a dummy row in order to make the 5 players on each time have the same striping.

* We should get rid of all the <hr> tags we used to separate the ads out with.

* Whitespace on multikills/streaks: Not intentional.  Not sure why the column is so big--we didn't set any explicit styling on it.  To fix maybe we have to set a fixed column width?

* Not really sure what is meant by "distinguish the graphs".  Also not sure about "distinguishing" the hero and played with tables.

* Opening a separate issue for the histogram bug
